src/app/projects/pages/autofill-project-form/autofill-project-form.component.ts

```diff
@@ -52,31 +51,56 @@ export class AutofillProjectFormComponent implements OnInit {
 
     if (this.publicationDoiCtrl.value) {
       const doi = this.publicationDoiCtrl.value;
-      this.doesDoiExist(doi).subscribe(doiExists => {
-          if (doiExists) {
-            this.alertService.error('This doi has already been used. Please contact our wranglers for further assistance', '');
-            return;
-          }
-          const params = {
-            [Identifier.DOI]: doi
-          };
-          this.router.navigate(['/projects', 'register'], {queryParams: params});
+      this.doesProjectWithDoiExist(doi).pipe(
+        filter(projectExists => projectExists === false),
+        switchMapTo(this.doesDoiExist(doi)),
+        filter(doiExists => doiExists)
+      ).subscribe(() => {
+          this.createProject(doi);
         },
         error => {
           this.alertService.error('An error occurred', error.message);
         });
     }
   }
 
-  doesDoiExist(doi: string): Observable<boolean> {
+  doesProjectWithDoiExist(doi: string): Observable<boolean> {
     const query = [];
     const criteria = {
       'field': 'content.publications.doi',
       'operator': 'IS',
       'value': doi
     };
     query.push(criteria);
-    return this.ingestService.queryProjects(query).pipe(map(data => !!data.page.totalElements));
+    return this.ingestService.queryProjects(query).pipe(
+      map(data => !!data.page.totalElements),
+      tap(projectExists => {
+        if (projectExists) {
+          this.alertService.error('This DOI has already been used. Please contact our wranglers for further assistance', '');
```

Comment:
Have you considered pointing the user to the project that uses this doi?

Reply:
Great Idea. Will work on it now.